src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala

```diff
@@ -1342,7 +1342,17 @@ class FileServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers with S
 
   private def mockCustomMetadataValuesResponse(customMetadataMock: CustomMetadataPropertiesRepository): ScalaOngoingStubbing[Future[Seq[FilepropertyvaluesRow]]] = {
     val staticMetadataRows = staticMetadataProperties.map(staticMetadata => {
```

Comment:
I'd rename this function to: `defaultMetadataRows` and replace `static` with default within the function